frontend/src/app/modules/work-space/work-space/work-space.component.ts

```diff
@@ -44,6 +44,10 @@ export class WorkSpaceComponent extends BaseComponent implements OnInit {
   }
 
   getGroups() {
+    if (this.user === null) {
```

Comment:
An empty user shouldn't be here

Reply:
no, it was, the problem was that user was null

Reply:
I checked it in this way: register, drop db and refresh page, the user was null as a result, so log out